cmd/ocp.go

```diff
@@ -32,8 +32,6 @@ import (
 //go:embed config/*
 var ocpConfig embed.FS
 
-const configDir = "config"
```

Comment:
I think its okay to keep this constant here itself. It is anyways not being used in other files.

Reply:
I moved it to common.go since it's referenced in index.go